ntbs-service/Migrations/20200422211917_AddPCTtoPostcodesTable.cs

```diff
@@ -0,0 +1,68 @@
+﻿using System;
+using System.Globalization;
+using System.IO;
+using CsvHelper;
+using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace ntbs_service.Migrations
+{
+    public partial class AddPCTtoPostcodesTable : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.Sql("ALTER TABLE [ReferenceData].PostcodeLookup ADD PCT varchar(10)");
+            
+            var pathToFile = Path.Combine(Environment.CurrentDirectory, "Models/SeedData/PostcodePCT.csv");
```

Comment:
Hmm, I'm worried that we're creating more of a mess for ourselves by having multiple source files to deal with postcodes. I would be tempted to bring this data into the `postcode.csv` file that already exists..?